bookList.js

```diff
@@ -0,0 +1,27 @@
+class Booklist {// eslint-disable-line no-unused-vars
+  constructor() {
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+  }
+
+  // function to add books
```

Comment:
- [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.